hc/api/tests/test_pause.py

```diff
@@ -9,10 +9,12 @@ def test_it_works(self):
         check.save()
 
         url = "/api/v1/checks/%s/pause" % check.code
-        r = self.client.post(url, "", content_type="application/json",
+        response = self.client.post(url, "", content_type="application/json",
                              HTTP_X_API_KEY="abc")
 
         ### Assert the expected status code and check's status
+        assert response.status_code == 200
```

Comment:
maintain consistence in coding style
assert response.status_code == 200 or self.assertEqual(response.status_code, 405)
stick to one style of assertion